packages/docusaurus-theme-classic/src/theme/Toggle/index.tsx

```diff
@@ -31,6 +31,10 @@ const ToggleComponent = memo(
     const [focused, setFocused] = useState(false);
     const inputRef = useRef<HTMLInputElement>(null);
 
+    useEffect(() => {
```

Comment:
hmm I have mitigated feeling because we now have the toggle state in 2 places: localstorage + react state

but it's probably fine because localstorage can't be considered reliable in all cases (and we want the toggle to keep working on such cases)

that should be fine for now 👍  but I have some refactoring ideas for later